model/move/doc.go

```diff
@@ -223,16 +224,19 @@ func (e *ExportDoc) CleanPreviousExports(archiver Archiver) error {
 func prepareExportDoc(i *instance.Instance, opts ExportOptions) *ExportDoc {
 	createdAt := time.Now()
 
-	// The size of the buckets can be specified by the options. If it is not
-	// the case, it is computed from the disk usage. An instance with 4x more
-	// bytes than another instance will have 2x more buckets and the buckets
-	// will be 2x larger.
-	bucketSize := opts.PartsSize
-	if bucketSize < minimalPartsSize {
-		bucketSize = minimalPartsSize
-		if usage, err := i.VFS().DiskUsage(); err == nil && usage > bucketSize {
-			factor := math.Sqrt(float64(usage) / float64(minimalPartsSize))
-			bucketSize = int64(factor * float64(bucketSize))
+	var bucketSize int64
+	if opts.LocalPath == "" {
+		// The size of the buckets can be specified by the options. If it is not
+		// the case, it is computed from the disk usage. An instance with 4x more
+		// bytes than another instance will have 2x more buckets and the buckets
+		// will be 2x larger.
```

Comment:
We should keep the system of buckets even when doing the export from the CLI.

Reply:
I didn't quite understand how the bucket system worked. This now fixed now.